tests/conftest.py

```diff
@@ -1,37 +1,92 @@
 import json
 from datetime import datetime, timezone
+from unittest.mock import MagicMock, patch
 
+import boto3
 import pytest
+from botocore.stub import Stubber
+
+# Usage and cost data fixtures for test_cost.py
 
 
 @pytest.fixture(scope="function")
 def input_data_usage():
-    with open("tests/test_data_usage.json") as f:
+    with open("tests/data/test_data_usage.json") as f:
         data = json.load(f)
     return data
 
 
 @pytest.fixture(scope="function")
 def input_data_cost():
-    with open("tests/test_data_cost.json") as f:
+    with open("tests/data/test_data_cost.json") as f:
         data = json.load(f)
     return data
 
 
 @pytest.fixture(scope="function")
 def output_data_hub():
-    with open("tests/test_output_hub.json") as f:
+    with open("tests/data/test_output_hub.json") as f:
         data = json.load(f)
     return data
 
 
 @pytest.fixture(scope="function")
 def output_data_component():
-    with open("tests/test_output_component.json") as f:
+    with open("tests/data/test_output_component.json") as f:
         data = json.load(f)
     return data
 
 
+# Usage and cost data for test_integration.py
+
+
+@pytest.fixture(scope="function", params=["compute", "home_storage"])
+def mock_usage_response(request):
+    """
+    Mock Prometheus response for compute and home storage components.
+    """
+    with patch("requests.get") as mock_get:
```

Comment:
I feel this patch is a bit too broad as it will cover all requests.get calls. How about we explicitly patch the one used in `query_usage` instead?
```suggestion
    with patch("src.jupyterhub_cost_monitoring.query_usage.requests.get") as mock_get:
```

Reply:
Ohh nice, I'll check that out!